### ISSUE TITLE
Comments are no longer supported in .env file

### ISSUE THREAD

**Issue by jstlaurent** [react-native-config/react-native-config#520]
As of version 1.4.1, comments (lines starting with `#`) are no longer supported in the parsed .env file.

On iOS, changes introduced in [0c42216](https://github.com/luggit/react-native-config/commit/0c42216725fd686b93900038e089fd71f9f75424) means that these lines will cause a **Invalid entry in .env file. Please verify your .env file is correctly formatted.** error to be raised.

I'm not building for Android, but I suspect the same will happen.

If that's the desired new behavior, I would suggest adding a section to the documentation mentioning it.

**Comment by Veid** [comment]
New lines are also no longer supported.

**Comment by gwmccull** [comment]
I'd really like to be able to have comments and new lines in the .env files. We have some keys that are commented out by default unless you're testing a very specific function, in which case, you uncomment them

**Comment by shllg** [comment]
Wrote a possible solution here: https://github.com/luggit/react-native-config/issues/522

**Comment by HolySamosa** [comment]
I agree with @gwmccull.  For our development .env, we use commented out lines extensively to switch between different development configurations.

Also in our production and staging environments, comments are critical for documentation.
